vllm/model_executor/model_loader/weight_utils.py

```diff
@@ -366,16 +366,22 @@ def filter_files_not_needed_for_inference(
 _BAR_FORMAT = "{desc}: {percentage:3.0f}% Completed | {n_fmt}/{total_fmt} [{elapsed}<{remaining}, {rate_fmt}]\n"  # noqa: E501
 
 
+def enable_tqdm(disable_progress_bar: bool):
+    return not disable_progress_bar and not torch.distributed.is_initialized(
+    ) or torch.distributed.get_rank() == 0
```

Comment:
Not sure if I understand the intention but: if `disable_progress_bar = True` means no progress bar then we could have the following scenario:
If progress bar is disabled and distributed mode is off, then we will evaluate the right side (get_rank), but in this case we're not in a distributed environment. Which then might raise an error

```suggestion
def enable_tqdm(disable_progress_bar: bool):
    if disable_progress_bar:
        return False
    if not torch.distributed.is_initialized():
        return True
    return torch.distributed.get_rank() == 0
```

Reply:
This is based on previous logics.

Additionally, we will just gated it with `disable_progress_bar` here.